src/templates/drizzle/nav.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="drizzle-Layout-nav drizzle-Nav" id="nav">
   <div class="drizzle-Nav-header">
-    <a href="/">
+    <a href="/" class="drizzle-u-flex drizzle-u-flexAlignItemsCenter">
```

Comment:
Remember to ignore a lot of my `role="presentation"` stuff if you go that route!